cat-tinder-frontend/src/components/Footer.test.js

```diff
@@ -0,0 +1,26 @@
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import Footer from './Footer'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()}) 
+
+describe("When the Footer renders", ()=> {
+    it("shows 4 NavItems", ()=> { 
+        const renderedFooter= shallow(<Footer/>)
+
+        const renderedNavItems = renderedFooter.find("NavItem")
+
+        expect(renderedNavItems.length).toEqual(4)
```

Comment:
I might be mistaken, I only see 3 NavItems in your Footer.  Does this test pass?